### PR TITLE
Use denali

### PR DESCRIPTION
**Pull request by jkusa**
## Description

Use denali

## Proposed Changes

- Add @denali/ember & ember-cli-sass-less to app & directory
- Comvert some components to denali components
- Convert some less files to sass
- Remove navi app landing page
- Update dependencies
- Fix dashboard filters alignment

## Screenshots

Screen Shot 2020-12-08 at 9 39 21 PM

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by kevinhinterlong — COMMENTED** [review]
Looks like you need to delete `dir-icon-test`. Should we add `@denali-design/ember`, `ember-cli-sass-less`, `sass` to every package? It seems like it's working except for tooltips so maybe we could ask the denali team to require a non-empty `data-tooltip` attr

File: packages/directory/tests/index.html [hunk 1]
```diff
@@ -1,33 +1,30 @@
...
-    {{content-for "head"}}
-    {{content-for "test-head"}}
+    {{content-for "head"}} {{content-for "test-head"}}
 
-    <link rel="stylesheet" href="{{rootURL}}assets/vendor.css">
-    <link rel="stylesheet" href="{{rootURL}}assets/dummy.css">
-    <link rel="stylesheet" href="{{rootURL}}assets/test-support.css">
+    <link rel="stylesheet" href="{{rootURL}}assets/vendor.css" />
+    <link rel="stylesheet" href="{{rootURL}}assets/dummy.css" />
+    <link rel="stylesheet" href="{{rootURL}}assets/navi-app.less.css" />
```
[the remark this answers is not in the document]
Comment (jkusa) [reply packages/directory/tests/index.html:14]: nice catch 👍

**Comment by jkusa** [comment]
> It seems like it's working except for tooltips so maybe we could ask the denali team to require a non-empty data-tooltip attr

The tooltip issue will be resolved in Denali this week. https://github.com/denali-design/denali-css/issues/110

I think we can get this in and upgrade shorlyt afterwords.

File: packages/app/app/components/top-bar.hbs [hunk 2]
```diff
@@ -1,13 +1,9 @@
...
-    </LinkTo>
-  </div>
-  <div class="top-bar__right">
-    <NaviIcon class="top-bar__user-icon" @icon="user" />
-    <span class="top-bar__user">{{this.loggedInUser}}</span>
-  </div>
-</div>
+<DenaliNavbar as |Nav|>
+  <Nav.Left as |Left|>
+    <Left.Logo src="{{this.rootURL}}img/navi-graph.svg"/>
```
[the remark this answers is not in the document]
Comment (jkusa) [reply packages/app/app/components/top-bar.hbs]: Done

**Review by kevinhinterlong — APPROVED** [review]

**Pull request merged**